File: 0389-find-the-difference/0389-find-the-difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
       long long sum1=0, sum2=0;
        for(int i=0;i<s.length();i++){
            sum1=sum1+s[i];
        }
        for(int j=0;j<t.length();j++){
            sum2=sum2+t[j];
        }
        return (char)(sum2-sum1);
    }
};

File: 0389-find-the-difference/0389-find-the-difference.rs
impl Solution {
    pub fn find_the_difference(s: String, t: String) -> char {
        let mut sum1: u8 = 0;
        let mut sum2: u8 = 0;
        for i in s.chars() {
            sum1+= i as u8;
        }
        for i in t.chars() {
            sum2+= i as u8;
        }
        return (sum2 - sum1) as char;
    }
}